apps/web/pages/_app.tsx

```diff
@@ -10,7 +10,7 @@ import NProgress from 'nprogress';
 import { Child } from '@app/types';
 import '../theme/style.css';
 import { AuthContext, AuthGuard } from '@app/context/Auth';
-
+import { Analytics } from '@vercel/analytics/react';
```

Comment:
analytics componsnet should be added in both return component

also I think you missed package.json file from apps directory make sure that you have committed that as well

@Surya-n

Reply:
also add rm -rf node_modules pnpm-lock.yaml

in    the workflow file